test/grpc.ts

```diff
@@ -215,7 +215,8 @@ describe('grpc', () => {
     const grpcClient = gaxGrpc();
 
     it('should load the test file', () => {
-      const protos = grpcClient.loadProto(TEST_PATH, TEST_FILE);
+      // tslint:disable-next-line:no-any
```

Comment:
what's up with the cast?

Reply:
Oops, must have been an artifact of the old implementation. Let me change that